pkg/prometheus/resource_selector.go

```diff
@@ -1089,6 +1101,10 @@ func (rs *ResourceSelector) validateKumaSDConfigs(ctx context.Context, sc *monit
 		if err := validateProxyConfig(ctx, config.ProxyConfig, rs.store, sc.GetNamespace()); err != nil {
 			return fmt.Errorf("[%d]: %w", i, err)
 		}
+
+		if err := validateServer(config.Server); err != nil {
```

Comment:
May be move this as first validation as there is more chance host can be wrong so that we don't need to do other validations

Reply:
Thanks. Done.